apps/chat/src/components/Marketplace/ApplicationDetails/ApplicationFooter.tsx

```diff
@@ -58,20 +64,23 @@ export const ApplicationDetailsFooter = ({
             className="shrink-0 text-accent-primary md:hidden [&_path]:fill-current"
             size={24}
           /> */}
-          {(isMyAppsTab || isMyApp) && (
-            <Tooltip tooltip={isMyApp ? t('Delete') : t('Remove')}>
-              <button
-                onClick={() => (isMyApp ? onDelete(entity) : onRemove(entity))}
-                className="group flex size-[34px] items-center justify-center rounded text-secondary hover:bg-accent-primary-alpha hover:text-accent-primary"
-                data-qa="application-edit"
-              >
-                <IconTrashX
-                  size={24}
-                  className="shrink-0 group-hover:text-accent-primary"
-                />
-              </button>
-            </Tooltip>
-          )}
+          {(isMyAppsTab || isMyApp) &&
+            installedModelIds.has(entity.reference) && (
```

Comment:
do we need this?

Reply:
If I remove the condition, then in the recommendations section when I open the card the remove button will appear, but it should not appear until it is used